File: DAYTWO/PYTHON/LEVEL_ONE/task_ten.py
average_of_numbers = 0;
sum_of_numbers = 0;
count = 0;
for loop in range(3):
	user_input = int(input("Enter your number: "))
	sum_of_numbers += user_input
	count += 1;
	average_of_numbers = sum_of_numbers / count

print("The average is: " , average_of_numbers)

File: DAYTWO/JAVA/LEVEL_ONE/TaskTen.java
import java.util.Scanner;

public class TaskTen{
public static void main(String... args){
Scanner input = new Scanner(System.in);

int averageOfnumbers = 0;
int sumOfNumbers = 0;
int count = 0;

int index = 0;
for (;index < 3; index++){
System.out.println("Enter your number");
int userInput = input.nextInt();
count += 1;
sumOfNumbers += userInput;
averageOfnumbers = sumOfNumbers / count;
}
System.out.println("The average is: " + averageOfnumbers);
}
}
